docs/DeveloperGuide.md

```diff
@@ -401,19 +952,134 @@ Use case ends.
 
 > A contact detail that is not meant to be shared with others
 
+##### switch
+
+> A functionality of certain `list` commands<br>
+  All the tasks added after switching will be automatically added to the current list (favourite) unless otherwise stated
+
+##### unswitch
+
+> A functionality of certain `list` commands<br>
+  After switching back to main, the tasks added will not be automatically added to this list (favourite)
+
 ## Appendix E : Product Survey
 
-**Product Name**
+**Wunderlist**
```

Comment:
Pretty good survey.

Question to ponder: any pros/cons from existing products helped in defining "deuce"?